tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "strictNullChecks": true,
     "skipLibCheck": true,
     "suppressImplicitAnyIndexErrors": true,
+    "ignoreDeprecations": "5.0",
```

Comment:
Why are we changing this?

Reply:
I got this error when deleting this line:

> Option 'suppressImplicitAnyIndexErrors' is deprecated and will stop functioning in TypeScript 5.5. Specify compilerOption '"ignoreDeprecations": "5.0"' to silence this error.ts
> Suppress noImplicitAny errors when indexing objects that lack index signatures.

Reply:
That's because your VSCode is configured for TypeScript v5+. Our project uses TypeScript v4.9 so it is not an issue.